### PR TITLE
SV-25 Create Button

### PR DESCRIPTION
**Pull request by CHAMPMOON**

### CODE REVIEW

File: src/components/common/button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    label: string;
+};
+  
+export const Button: React.FC<ButtonProps> = ({onClick, id, label}) => {
+return (
+    <button className="btn primary btn-sm" onClick={onClick} id={id}>
+    {label}
+    </button>
+);
+}
```
Comment (osovv) [inline src/components/common/button/Button.tsx:15]: add new line

Comment (osovv) [reply src/components/common/button/Button.tsx:15]: vscode settings
```
"editor.insertSpaces": true,
"editor.formatOnSave": true,
```

**Review by osovv — CHANGES_REQUESTED** [review]
remove `package-lock.json` and `yarn.lock`
image

**Review by CHAMPMOON — COMMENTED** [review]
all done

**Review by osovv — APPROVED** [review]

**Pull request merged**